src/test/resources/features/dcu/DCUDataInput.feature

```diff
@@ -90,55 +116,79 @@ Feature: DCU Data Input
+    And I load and claim the current case
     And I click the "Add" button on the "ADD MEMBER OF PARLIAMENT" page
     Then an error message should be displayed as I must select a member of parliament from the drop down
 
   @Validation
   Scenario: User must select a correspondent type from the drop down if the correspondent is not an MP
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the "Add" button on the "RECORD CORRESPONDENT DETAILS" page
     Then an error message should be displayed as I have not selected the correspondent type
 
   @Validation
   Scenario: User must enter text in correspondent's Full Name field
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the "Add" button on the "RECORD CORRESPONDENT DETAILS" page
     Then an error message should be displayed as I have not entered text in the full name field
 
   @Validation
   Scenario: User must enter text in the text box when creating a Case note at the Data Input stage
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the add button when creating a case note
     Then an error message should be displayed as I have not entered text in the Case Note text box
 
   @OtherTests
   Scenario: User removes a correspondent
-    When I fill all mandatory fields on the "Data Input" page with valid data
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
+    And I fill all mandatory fields on the "Data Input" page with valid data
     And I click the "Continue" button
     And I add "Nicola Sturgeon" MP as a correspondent
     And I remove the primary correspondent
     Then there shouldn't be a primary correspondent displayed
 
   @OtherTests
   Scenario: User edits an existing correspondent
-    When I fill all mandatory fields on the "Data Input" page with valid data
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
+    And I fill all mandatory fields on the "Data Input" page with valid data
     And I click the "Continue" button
     And I add "Nicola Sturgeon" MP as a correspondent
     And I edit the primary correspondents name
     Then the correspondents name should be updated
 
   @DCURegression
-  Scenario: User creates a MIN case and checks that the stage deadlines are correct
-    Then the stage deadline dates for a "MIN" case are correct
+  Scenario Outline: User creates a DCU case and checks that the stage deadlines are correct
+    When I create a "<caseType>" case and move it to the "Data Input" stage
+    And I load and claim the current case
+    Then the stage deadline dates for a "<caseType>" case are correct
+    Examples:
+      | caseType  |
+      | MIN       |
+      | TRO       |
 
```

Comment:
Again, I dont think this requires parameterisation as this isnt strictly a workflow test. Does this feature not have a @DCUWorkflow tagged scenario that tests sending a case from Data Input to Markup stage? Thats the one I thought we would alter